spec/features/welcome/index_spec.rb

```diff
@@ -12,6 +12,10 @@
   end
 
   it 'can log in and out of the application with a valid Google token', :vcr do
+    allow(BackEndFacade).to receive(:get_user_friends).with(@user.id).and_return([])
+    allow(BackEndFacade).to receive(:get_user_gyms).with(@user.id).and_return([])
+    allow(BackEndFacade).to receive(:get_user_events).with(@user.id).and_return([])
```

Comment:
These stubs needed to be added to get the welcome index spec to pass.

Reply:
Added on my branch, same problem